src/components/question/_macro.njk

```diff
@@ -11,41 +11,76 @@
         {% if params.readDescriptionFirst is defined and params.readDescriptionFirst == true %} 
             <div aria-hidden="true" {% if params.legendIsQuestionTitle is not defined %}class="ons-question__description"{% endif %}>{{ params.description | safe }}</div>
         {% elif params.legendIsQuestionTitle is not defined %}
-            <div class="ons-question__description">{{ params.description | safe }}</div>
+            <div class="ons-question__description ons-u-mb-m">{{ params.description | safe }}</div>
         {% else %}
             {{ params.description | safe }}
         {% endif %}
     {% endset %}
 
     {% set instHtml %}
-        <div class="ons-question__instruction ons-u-mb-s">{{ params.instruction | safe }}</div>
+        <div class="ons-question__instruction ons-u-mb-m">{{ params.instruction | safe }}</div>
+    {% endset %}
+
+    {% set questionDefinition %}
+        {% from "components/collapsible/_macro.njk" import onsCollapsible %}
+        {% call onsCollapsible({
+            "id": params.definition.id,
+            "titleTag": 'h2',
+            "classes": 'ons-u-mb-m',
+            "title": params.definition.title
+        }) %}
+            {% if params.definition.content %}
+                {{ params.definition.content | safe }}
+            {% endif %}
+        {% endcall %}
+    {% endset %}
+
+    {% set questionGuidance %}
+        {% from "components/panel/_macro.njk" import onsPanel %}
+        {% call onsPanel({
+            "classes": "ons-u-mb-m"
+        }) %}
+            {% if params.guidance.content %}
+                {{ params.guidance.content | safe }}
+            {% endif %}
+            {%- for item in params.guidance.lists -%}
+                {% if item.listHeading %}
+                    <h2 class="ons-u-fs-r--b">{{ item.listHeading }}</h2>
+                {% endif %}
+                {% if item.listLeadingLine %}
+                    <p>{{ item.listLeadingLine }}</h2>
+                {% endif %}
+                {% from "components/lists/_macro.njk" import onsList %}
+                {{
+                    onsList({
+                        "itemsList": item.itemsList
+                    })
+                }}
+            {% endfor -%}
```

Comment:
```suggestion
            {% if params.guidance.lists is defined and  params.guidance.lists %}
                {%- for item in params.guidance.lists -%}
                    {% if item.listHeading is defined and item.listHeading %}
                        <h2 class="ons-u-fs-r--b">{{ item.listHeading }}</h2>
                    {% endif %}
                    {% if item.listLeadingLine is defined and item.listLeadingLine %}
                        <p>{{ item.listLeadingLine }}</h2>
                    {% endif %}
                    {% from "components/lists/_macro.njk" import onsList %}
                    {{
                        onsList({
                            "itemsList": item.itemsList
                        })
                    }}
                {% endfor -%}
            {% endif -%}
```

Should probably check here if guidance.lists exists as its not mandatory and add is defined in a couple of places